### PR TITLE
start lesson

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by SSalekin** [comment]
ok

**Comment by sujoydatta** [comment]
ok

File: app/controllers/lessons_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class LessonsController < ApplicationController
+  before_action :load_lesson, only: [:edit]
+
+  def new
+  end
+
+  def create
+    @lesson = Lesson.new lesson_params
+    @lesson.attributes = {user_id: current_user.id}
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: `@lession.user = current_user`

File: app/controllers/lessons_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class LessonsController < ApplicationController
+  before_action :load_lesson, only: [:edit]
+
+  def new
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: i think we no need this action

File: app/controllers/lessons_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class LessonsController < ApplicationController
+  before_action :load_lesson, only: [:edit]
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: `cancancan`, use it

**Comment by sujoydatta** [comment]
ok

**Pull request merged**